src/encoded/static/components/matrix_reference_epigenome.js

```diff
@@ -415,6 +411,92 @@ MatrixHeader.contextTypes = {
     navigate: PropTypes.func,
 };
 
+const ClickableThumbnail = (props) => {
+    const expandedThumbnail = props.expandedThumbnail;
```

Comment:
should this be called "expand"?

Reply:
I've renamed "expandedThumbnail" to be "isThumbnailExpanded", hopefully this is clearer.